spec/package_json_spec.rb

```diff
@@ -515,4 +523,55 @@
       end
     end
   end
+
+  describe "the private method #determin_yarn_version" do
```

Comment:
As I said in my comment elsewhere, please don't use `send` as its overly brittle - see [here](https://github.com/shakacode/package_json/pull/7#discussion_r1341987054)